assets/app/view/game/round/draft.rb

```diff
@@ -85,9 +86,31 @@ def render_input
         def render_player
           return nil if @hidden
 
-          h(:div, [
-            h(Player, player: @player, game: @game),
-          ])
+          draft_dist = @round.steps[-1]
+          drafted = draft_dist.choices[@player]
+
+          player = @player.clone
```

Comment:
i'm not a big fan of this approach, it seems a bit hacky, can we instead do a simpler view?

Reply:
just listing the companies name that have been drafted and the remaining money so we don't have to clone and mock objects?